test/src/main/java/org/apache/rocketmq/test/listener/AbstractListener.java

```diff
@@ -95,6 +95,28 @@ public void stopRecv() {
         return sendMsgs;
     }
 
+    public long waitForMessageConsume(int size,
+        int timeoutMills) {
+
+        long curTime = System.currentTimeMillis();
+        while (true) {
+            if (msgBodys.getDataSize() >= size) {
+                break;
+            }
+            if (System.currentTimeMillis() - curTime >= timeoutMills) {
+                logger.error(String.format("timeout but  [%s]  not recv all send messages!",
```

Comment:
but  [%s]  not recv all send messages! --> but [%] did not receive all messages sent!